File: src/main/org/nlogo/prim/etc/_nodisplay.java
package org.nlogo.prim.etc ;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Syntax;

public final strictfp class _nodisplay
	extends Command
{
	@Override
	public void perform( final org.nlogo.nvm.Context context )
	{
		world.displayOn( false ) ;
		context.ip = next ;
	}
	@Override
	public Syntax syntax()
	{
		int[] right = {} ;
		return Syntax.commandSyntax( right ) ;
	}
}

File: headless/src/main/org/nlogo/prim/etc/_nodisplay.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context }

class _nodisplay extends Command {
  override def syntax =
    Syntax.commandSyntax
  override def perform(context: Context) {
    world.displayOn(false)
    context.ip = next
  }
}
